docs/concept/TAP7.1-solution-design.md

```diff
@@ -0,0 +1,425 @@
+| 1   | Developers<ul><li>External Developers (not Tractus-X developers): Utilize the chart for demonstration or evaluation purposes.</li><li>Experienced software and product developers who are developing a business app for a CATENA-X data space. They need to develop the app against the interfaces of the data space and integrate them accordingly.</li><li>They emphasize efficient and cost-effective implementation:<ul><li>Cost Efficiency: They do not want to use expensive SaaS solutions.</li><li>Technical Independence: They lack the knowledge and capacity to deploy the enablement and core services of the data space themselves.</li><li>Focus on Development: They concentrate on application development without delving deeply into the deployment and operation of the underlying infrastructure.</li></ul></li><li>Out of Scope: Developers running tests in test runners are not considered within this scope.</li></ul> | Data Consumer <br/> Data Provider <br/> Enablement Service Provider |                                                                                                                                                                                                                                                                                                         | <ul><li>Needs / sources of information<ul><li><a href="https://eclipse-tractusx.github.io/Kits/">Kit documentation tractus-x</a> for interaction purposes</li><li><a href="https://catenax-ev.github.io/">Catena-X doucmentation</a> for standards and regulations</li><li><a href="https://github.com/eclipse-tractusx/tractus-x-umbrella">Umbrella documentation</a> for umbrella component information</li></ul></li><li>Challenges → Solution<ul><li>Decentralization e.g. Where are other systems? How to interact with them? → Solution: Umbrella charts and bundles for deployment</li><li>Complex setup → Solution: delivery of small capability bundles</li></ul></li></ul> |
```

Comment:
Please avoid HTML Tags as these are not TRG conform

Reply:
Unfortunately, that is the only possibility to include a list in the table.